static/css/ibm-plex-sans-var.css

```diff
@@ -0,0 +1,17 @@
+@font-face {
+    font-family: 'IBM Plex Sans';
+    font-style: normal;
+    font-display: swap;
+	src:url('../fonts/IBMPlexSansVar-Roman.woff2') format('woff2 supports variations'),
+        url('../fonts/IBMPlexSansVar-Roman.woff2') format('woff2-variations');
+	font-weight: 100 700;
```

Comment:
We are also using font-weight `200-300-500-600` do we need to install them and import/define them here?

Reply:
> We are also using font-weight `200-300-500-600` do we need to install them and import/define them here?

No, `font-weight: 100 700;` is a range. it means all weights from 100 to 700. (they don't even need to be multiple of 100, for example, we can have: `font-weight: 456;`)

Reply:
I see. Thank you for the clarification 😄🙏🏻